core/src/Kokkos_CopyViews.hpp

```diff
@@ -1327,15 +1327,14 @@ inline void contiguous_fill(
 }
 
 // Default implementation for execution spaces that don't provide a definition
-template <typename ExecutionSpace, class DT, class... DP>
+template <typename ExecutionSpace, class ViewType>
 struct ZeroMemset {
-  ZeroMemset(const ExecutionSpace& exec_space, const View<DT, DP...>& dst,
-             typename ViewTraits<DT, DP...>::const_value_type& value) {
+  ZeroMemset(const ExecutionSpace& exec_space, const ViewType& dst,
+             typename ViewType::const_value_type& value) {
```

Comment:
@masterleinad Am I correct that the default version should also have template 
```
template <typename ExecutionSpace, class ViewType>
```
If so, in Kokkos is it equivalent to use 
```
View<DT, DP...>::const_value_type <-> ViewTraits<DT, DP...>::const_value_type
```

Reply:
Yes, `View<DT, DP...>::const_value_type` and `ViewTraits<DT, DP...>::const_value_type` are the same.